main.py

```diff
@@ -1,10 +1,45 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for player_data in data:
+        race_data = data[player_data]["race"]
+        guild_data = data[player_data]["guild"]
+        skills_data = race_data["skills"]
+
+        race, _ = Race.objects.get_or_create(
+            name=race_data["name"],
+            defaults={"description": race_data.get("description", "")}
+        )
+        print(race)
+        if guild_data is not None:
+            guild, _ = Guild.objects.get_or_create(
+                name=guild_data["name"],
+                defaults={"description": guild_data.get("description", "")}
+            )
+        else:
+            guild = None
```

Comment:
You can create this viriable at the start and do not use else block .

If your condition is triggered, the contents of the variable will simply be overwritten